sdcm/stress_thread.py

```diff
@@ -136,10 +136,27 @@ def create_stress_cmd(self, node, loader_idx, keyspace_idx):
 
     @staticmethod
     def _add_errors_option(stress_cmd: str, to_add: list):
```

Comment:
We can add a return type hint:
```suggestion
    def _add_errors_option(stress_cmd: str, to_add: list) -> str:
```